docs/source/zarr.rst

```diff
@@ -0,0 +1,11 @@
+Zarr
+====
+
```

Comment:
I think it would be good to give a brief brief mention of what zarr is:

Zarr is a binary file format for chunked, compressed, N-Dimensional array.  It is used throughout the PyData ecosystem and especially for climate and biological science applications. 

Something like that ?

Reply:
Sound good, added.